admin/common_auth/admin.py

```diff
@@ -0,0 +1,54 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from django.contrib.auth.models import Permission
+from django.contrib.auth.forms import PasswordResetForm
+from .models import MyUser
+from forms import CustomUserRegistrationForm
+
+from models import OsfUser
```

Comment:
This import is coming from the same module as .models.MyUser on line 5. These could be combined just for ease of merging later.